### DISCUSSION TITLE
Module changes in interactive window are not working

### DISCUSSION THREAD

**Discussion by linette-zyy** [microsoft/pylance-release#3812] [Enhancement]
**Environment**
![image](https://user-images.githubusercontent.com/59192491/119962269-7237cc80-bfd9-11eb-9f5b-ce827b1c7f60.png)

**Steps to Reproduce**
1. Using module combo box, switch to io module
2. Define a new method: def hello(): pass
3. Check that it exists: dir()
4. Switch back to __main__ module
5. Check that it doesn't exist in __main__ module: dir()
6. import io

**Expected behavior**
io.hello() # you should get intellisense on hello

**Actual result**
Cannot find the new method defined.
![image](https://user-images.githubusercontent.com/59192491/119961867-19683400-bfd9-11eb-9d78-7b867f19cb21.png)

**Comment by rchiodo** [comment]
This would require forwarding the module changes to pylance. Not sure we want to do that? Pylance would think that edits had happened to the original file.

**Comment by greazer** [comment]
This is something that would need support from Pylance to support.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
